recipes/cWB/meta.yaml

```
@@ -0,0 +1,99 @@
+{% set name = "cWB" %}
+{% set version = "6.4.2" %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: https://gitlab.com/gwburst/public/library/-/archive/cWB-{{ version }}_test_conda_v2/library-cWB-{{ version }}_test_conda_v2.tar.gz
+   sha256: b5318a2aba3e029a3b4942d7a51d8878d6aa89dee88b34a5096a8de37dce7bad
+   patches:
+     # use rootcling from build env when cross-compiling
+     - rootcling-cross.patch  # [build_platform != target_platform]
+
+build:
```

```suggestion
build:
  run_exports:
    - {{ pin_subpackage( name|lower, max_pin='x.x.x' ) }}
```
Because this package is shipping shared libraries and headers, it needs a run_export. Above is a *suggestion*; please adjust this pinning expression according to the ABI compatibility guarantees of cWB.